### PR TITLE
Compliance CIS Section 1

### PR DESCRIPTION
**Pull request by lborguetti**
This PR enable compliance with CIS Section 1

### CODE REVIEW

File: molecule/cis-benchmark/verify.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@
         dest: "{{ goss_test_directory }}/{{ item | basename }}"
       with_fileglob:
         - "{{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }}/test_*.yml"
-        - "{{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }}/scripts/test_*/*.sh"
+        - "{{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }}/scripts/test_section_06_level1/*.sh"
```
Comment (evaldofelipe) [inline molecule/cis-benchmark/verify.yml:34]: Some reason to fix the section name in the path?

File: tasks/section_01_level1.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  tags: section01
+
+- name: ensure blacklist insecure kernel modules
+  copy:
+    src: "etc/{{ item }}"
+    dest: "/etc/{{ item }}"
+    mode: 0644
+    owner: root
+  with_items:
+    - modprobe.d/cis-section_01_level1.conf
```
Comment (evaldofelipe) [inline tasks/section_01_level1.yml:34]: I don't understand the reason to use a cis-section as file name here, to copy to VM to this `/etc/...` path. It's not can cause any issue inside a VM?

File: tasks/section_01_level1.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  tags: section01
+
+- name: ensure set a hard limit on core dumps
+  copy:
+    src: "etc/{{ item }}"
+    dest: "/etc/{{ item }}"
+    mode: 0644
+    owner: root
+  with_items:
+    - security/limits.d/cis-section_01_level1.conf
```
Comment (evaldofelipe) [inline tasks/section_01_level1.yml:44]: +1

File: tasks/section_01_level1.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- name: ensure copy sysctl configurations
+  copy:
+    src: "etc/sysctl.d/{{ item }}"
+    dest: "/etc/sysctl.d/{{ item }}"
+    mode: 0600
+    owner: root
+  register: sysctl
+  with_items:
+    - cis-section_01_level1.conf
```
Comment (evaldofelipe) [inline tasks/section_01_level1.yml:55]: +1

File: tasks/section_03_level1.yml [hunk 5]
```diff
@@ -9,11 +9,5 @@
...
     - cis-section_03_level1.conf
-  tags: sysctl
-
-- name: ensure apply sysctl configurations
-  systemd:
-    name: systemd-sysctl
-    state: "{{ (sysctl.changed) | ternary('restarted', 'started') }}"
-    enabled: true
-  tags: sysctl
+  notify: "restart sysctl"
```
Comment (evaldofelipe) [inline tasks/section_03_level1.yml:12]: This `notify:` is a alternative to restart and check the `sysctl` status?

File: molecule/cis-benchmark/verify.yml (hunk 1 above)
Comment (lborguetti) [reply molecule/cis-benchmark/verify.yml:34]: Yes! The oldest code don't expand the path :(

File: tasks/section_01_level1.yml (hunk 2 above)
Comment (lborguetti) [reply tasks/section_01_level1.yml:34]: This just is a reference to CIS Controls.

It's not cause any issue, any file plus suffix `.conf` is loaded at boot time.

File: tasks/section_01_level1.yml (hunk 3 above)
Comment (lborguetti) [reply tasks/section_01_level1.yml:44]: same above. =)

File: tasks/section_01_level1.yml (hunk 4 above)
Comment (lborguetti) [reply tasks/section_01_level1.yml:55]: same above. =0

File: tasks/section_03_level1.yml (hunk 5 above)
Comment (lborguetti) [reply tasks/section_03_level1.yml:12]: Yes. 

Take a look here : https://docs.ansible.com/ansible/2.5/user_guide/playbooks_intro.html#handlers-running-operations-on-change

File: tasks/section_01_level1.yml (hunk 2 above)
Comment (evaldofelipe) [reply tasks/section_01_level1.yml:34]: nice!

File: tasks/section_03_level1.yml (hunk 5 above)
Comment (evaldofelipe) [reply tasks/section_03_level1.yml:12]: nice tip! thanks 🤙

**Review by evaldofelipe — APPROVED** [review]
LGTM!

**Pull request merged**